providers/openstack/README.md

```diff
@@ -140,45 +171,61 @@ openstackclusterstackreleasetemplate.infrastructure.clusterstack.x-k8s.io/cspote
 
 ## Create the workload cluster resource (SCS-User/customer)
 
-Create and apply `cluster.yaml` file to the management cluster:
+To create a workload cluster you must configure the following things:
 
 ```bash
-cat <<EOF | kubectl apply -f -
+export CS_CLUSTER_NAME=cs-cluster
+export CS_POD_CIDR=192.168.0.0/16
+export CS_SERVICE_CIDR=10.96.0.0/12
+export CS_EXTERNAL_ID=ebfe5546-f09f-4f42-ab54-094e457d42ec # gx-scs
+export CS_CLASSNAME=openstack-"${CS_STACK_NAME}"-"${CS_KUBERNETES_VERSION/./-}"-"${CS_STACK_VERSION}"
+export CS_KUBERNETES_PATCH_LEVEL=3
```

Comment:
Also here just a naming suggestion to shorten the name of the variable, which could be `CS_K8S_PATCH_VERSION`

Reply:
fixed and squashed into 673337ceffa5b90ddf996572bca5ea4a26b28175